BurnOutSharp/ProtectionType/Macrovision.cs

```diff
@@ -86,10 +120,17 @@ public string CheckFilePath(string path)
         {
             // TODO: Add all common Macrovision file path checks here
 
+            // Run C-Dilla file checks
+            string cDilla = CDillaCheckFilePath(path);
```

Comment:
**Question:** Were you planning on doing a list of detected protections similar to the PE checks?